_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -4232,7 +4246,7 @@
 "aAe" = (
 /obj/machinery/computer/security{
 	dir = 1;
-	network = list("SS13","Research Outpost","Mining Outpost")
+	network = list("SS13","Research  Outpost","Mining  Outpost")
```

Comment:
I'm not sure what caused these extra spaces to appear, but please fix them, and on the lines below

Reply:
IIRC, it means they're on an outdated strongDMM version

Reply:
Superfluous spacing has been removed (I have also gone ahead and done the same on centcomm.dmm since it had the same issues.)